docs/jwt-vc-json-support.md

```
@@ -0,0 +1,183 @@
+
+If verification succeeds, the credential is saved to the Wallet's store.
+
+##### 8. Decode JWT, extract credentialSubject as fullResolvedPayload
+
+The raw JWT credential is decoded and the `credentialSubject` is extracted from `payload.vc.credentialSubject`. This is stored as `fullResolvedPayload` on the processed credential and is used for all subsequent field rendering.
+
+##### 9. Get credential issuer metadata for rendering
+
+The Wallet fetches the issuer's well-known configuration to determine how the credential should be displayed.
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)